src/Toolkit/Toolkit.Maui/BasemapGallery/ByteArrayToImageSourceConverter.cs

```diff
@@ -30,7 +30,7 @@ internal class ByteArrayToImageSourceConverter : IValueConverter
                 return ImageSource.FromStream(() => new MemoryStream(rawBuffer));
             }
 
-            return null;
```

Comment:
This seems like an odd fallback for a generic bytes to image source converter.

Reply:
It is internal and only used in BasemapGallery. There are MAUI issues when converters are used in collection views. Previous to this change I was seeing the wrong thumbnail shown for items after scrolling. With this change, the issue appears to be fixed.

Reply:
Could you add a comment there for future reference then?